packages/sentry-client/src/sentry/actions.ts

```diff
@@ -53,32 +53,43 @@ export const pollServer = (server: any) => {
 		let requestInit: RequestInit = {
 			method: "GET",
 			headers: new Headers(),
-			mode: "cors",
 			cache: "default"
 		};
 
-		fetch(server.url, requestInit).then((response: Response) => {
+		fetch(server.host, requestInit).then((response: Response) => {
 			// We care about the responseCode and the body
 
 			// Go ahead and set status to unavailable
 			let status = STATUS.OUTAGE;
 
-			// If responseCode is 200, elevate to issue
-			if (response.status === 200) {
-				status = STATUS.ISSUE;
-			}
-
 			// If response body is in expected format, read it
 			// Set to issue, availble, or maintainence accordingly
 
-			dispatch({
-				type: actionsTypes.POLL_SERVER,
-				payload: {
-					id: server.id,
-					status: status,
-					body: response.body,
-				}
-			});
+			// TODO: Refactor
```

Comment:
Easy refactor is generating a payload based on whether the response is ok and generalizing the `dispatch`